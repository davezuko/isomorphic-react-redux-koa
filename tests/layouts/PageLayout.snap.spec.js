import React from 'react'
import PageLayout from 'layouts/PageLayout/PageLayout'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer.create(<PageLayout />).toJSON()
  expect(tree).toMatchSnapshot()
})
