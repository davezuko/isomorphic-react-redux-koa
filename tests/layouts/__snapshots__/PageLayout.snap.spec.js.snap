// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container text-center"
>
  <h1>
    React Redux Starter Kit
  </h1>
  <a
    onClick={[Function]}
    style={Object {}}
  >
    Home
  </a>
   Â· 
  <a
    onClick={[Function]}
    style={Object {}}
  >
    Counter
  </a>
  <div
    className="page-layout__viewport"
  />
</div>
`;
